client-http/src/test/scala/com/cloudera/livy/client/http/LivyConnectionSpec.scala

```diff
@@ -60,31 +60,29 @@ class LivyConnectionSpec extends FunSpecLike with BeforeAndAfterAll with LivyBas
       csh
     }
 
-    def staticServlet(): HttpServlet = new HttpServlet {
-      override def doGet(req: HttpServletRequest, resp: HttpServletResponse): Unit = {
-        resp.getWriter.print("true")
-      }
-    }
-
-    def test(password: String): Unit = {
+    def test(password: String, livyConf: LivyConf = new LivyConf()): Unit = {
       val username = "user name"
 
-      val server = new Server(0)
-      val context = new ServletContextHandler(ServletContextHandler.SESSIONS)
-      context.setSecurityHandler(basicAuth(username, password, "realm"))
-      context.setContextPath("/")
-      context.addServlet(new ServletHolder(staticServlet()), "/")
-      server.setHandler(context)
+      val server = new WebServer(livyConf, "0.0.0.0", 0)
+      server.context.setSecurityHandler(basicAuth(username, password, "realm"))
+      server.context.setResourceBase("src/main/com/cloudera/livy/server")
+      server.context.setInitParameter(ScalatraListener.LifeCycleKey,
+        classOf[HttpClientTestBootstrap].getCanonicalName)
+      server.context.addEventListener(new ScalatraListener)
       server.start()
 
       val utf8Name = UTF_8.name()
-      val uri = new URIBuilder(server.getURI())
+      val uri = new URIBuilder()
+        .setScheme(server.protocol)
+        .setHost(server.host)
+        .setPort(server.port)
         .setUserInfo(URLEncoder.encode(username, utf8Name), URLEncoder.encode(password, utf8Name))
         .build()
       info(uri.toString)
       val conn = new LivyConnection(uri, new HttpConf(null))
       try {
-        conn.get(classOf[Boolean], "/") shouldBe true
+        conn.get(classOf[Object], "/") should not be (null)
```

Comment:
Because previously we use this [servlet](https://github.com/cloudera/livy/pull/331/files#diff-6ca014ee7958eb376596cb519952a394L63) which will return `Boolean`. Now we change to use `InteractiveSessionServlet`, so using `Boolean` will meet JSON parse error.